zebra-consensus/src/transaction/tests.rs

```diff
@@ -86,8 +86,34 @@ fn fake_v5_transaction_with_orchard_actions_has_inputs_and_outputs() {
     // guaranteed structurally by `orchard::ShieldedData`)
     insert_fake_orchard_shielded_data(&mut transaction);
 
-    // If a transaction has at least one Orchard shielded action, it should be considered to have
-    // inputs and/or outputs
+    // The check will fail if it haves no flags
```

Comment:
```suggestion
    // The check will fail if the transaction has no flags
```

Reply:
Thanks, i fixed this comment as part of https://github.com/ZcashFoundation/zebra/pull/2398/commits/05019a9d4d5b7a18517489029f9a9a19171b76c4